Dockerfile.special-resource-operator-build

```
@@ -0,0 +1,17 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.16
+
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+RUN chmod 700 get_helm.sh
+RUN ./get_helm.sh
+RUN yum install -y which
+RUN yum install -y podman docker
+RUN curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /tmp/kubectl
+RUN install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.1
+RUN GO111MODULE=on go install golang.stackrox.io/kube-linter/cmd/kube-linter@latest
```

```suggestion
RUN go install golang.stackrox.io/kube-linter/cmd/kube-linter@latest
```
`GO111MODULE` is already set line 3.

---

Skipper would then run all `make` commands in the container, right? I am not sure then if it's worth installing those dependencies manually; the Makefile has target that do it.

---

skipper is just a wrap for any command that user would like to run on Makefile. So, if previously user would run:
make local-image-build
now he will run:
skipper make local-image-build
Regarding installing dependencies:  since skipper is actually a running image, then installing dependencies via Makefile means that they will need to be installed on every skipper run. Installing them via Dockerfile means that they are present in the image all the tiime